docs/data/sdks/ios-swift/migration.md

```diff
@@ -242,175 +184,115 @@ The APIs for setting user properties are the same, except for the removal of `in
 
-=== "Amplitude-iOS"
-
-    ```swift
-    let userId = "TEST-USER-ID"
-    Amplitude.instance().setUserId(userId)
-    ```
-
-=== "Amplitude-Swift"
-
-    ```swift
-    let userId = "TEST-USER-NAME"
-    amplitude.setUserId(userId: userId)
-    ```
+```diff
+let userId = "TEST-USER-ID"
+- Amplitude.instance().setUserId(userId)
++ amplitude.setUserId(userId: userId)
+```
 
 #### `setDeviceId()`
 
 Set a device ID on `amplitude` without calling `instance()`.
 
-=== "Amplitude-iOS"
-
-    ```swift
-    let deviceId = "TEST-DEVICE-ID"
-    Amplitude.instance().setDeviceId(deviceId)
-    ```
-
-=== "Amplitude-Swift"
-
-    ```swift
-    let deviceId = "TEST-DEVICE-ID"
-    amplitude.setDeviceId(deviceId: deviceId)
-    ```
+```diff
+let userId = "TEST-DEVICE-ID"
+- Amplitude.instance().setDeviceId(userId)
++ amplitude.setDeviceId(deviceId: deviceId)
+```
 
 #### `setSessionId()`
 
 Set a session ID on `amplitude` without calling `instance()`.
 
-=== "Amplitude-iOS"
-
-    ```swift
-    let timestamp = Int64(NSDate().timeIntervalSince1970 * 1000)
-    Amplitude.instance().setSessionId(timestamp)
-    ```
-
-=== "Amplitude-Swift"
-
-    ```swift
-    let timestamp = Int64(NSDate().timeIntervalSince1970 * 1000)
-    amplitude.setSessionId(sessionId: timestamp)
-    ```
+```diff
+let timestamp = Int64(NSDate().timeIntervalSince1970 * 1000)
+- Amplitude.instance().setSessionId(timestamp)
++ amplitude.setSessionId(sessionId: timestamp)
+```
 
 #### `clearUserProperties()`
 
 The `clearUserProperties` API has been removed, but you can now use the unified `identify` API to remove user properties. 
 
-=== "Amplitude-iOS"
-
-    ```swift
-    Amplitude.instance().clearUserProperties()
-    ```
-
-=== "Amplitude-Swift"
-
-    ```swift
-    let identify = Identify()
-    identify.clearAll()
-    amplitude.identify(
-      identify: identify
-    )
-    ```
+```diff
+let timestamp = Int64(NSDate().timeIntervalSince1970 * 1000)
+- Amplitude.instance().clearUserProperties()
++ let identify = Identify()
++ identify.clearAll()
++ amplitude.identify(identify: identify)
+```
 
 #### `setUserProperties()`
 
 The `setUserProperties` API has been removed, but you can now use the unified `identify` API to add user properties. 
 
-=== "Amplitude-iOS"
-
-    ```swift
-    Amplitude.instance().setUserProperties([
-      "membership": "paid",
-      "payment": "bank",
-    ])
-    ```
-
-=== "Amplitude-Swift"
-
-    ```swift
-    let identify = Identify()
-    identify
-      .set(property: "membership", value: "paid")
-      .set(property: "payment", value: "bank")
-    amplitude.identify(identify: identify)
-    ```
+```diff
+- Amplitude.instance().setUserProperties([
+-   "membership": "paid",
+-   "payment": "bank",
+- ])
++ let identify = Identify()
++ identify
++   .set(property: "membership", value: "paid")
++   .set(property: "payment", value: "bank")
++ amplitude.identify(identify: identify)
```

Comment:
https://github.com/amplitude/Amplitude-Swift/blob/main/Sources/Amplitude/Amplitude.swift#L100 can be used instead of `setUserProperties()`

Reply:
Fixed.